docs/evaluate/major-components.mdx

```diff
@@ -19,15 +19,12 @@ Temporal has four major components:
 
 - **Temporal SDK** used to build applications and run applications.
 - **Temporal Service** that maintains the state of the applications.
-- **Temporal CLI** used to interact with the Temporal Service for a variety of purposes.
-- **Temporal UI** primary used to view the state of applications.
+- **Temporal CLI** used to interact with the Temporal Service.
+- **Temporal UI** primarily used to view the state of applications.
```

Comment:
I think the SDK is used to 
* provide language-specific APIs that allow your code to communicate and coordinate with Temporal Service orchestration.

I think the Service 
* is responsible for providing Namespace-specific task brokering,  event history, and persistence for Workflow execution. The Temporal Service orchestrates your Tasks by communicating with SDK-provided Client APIs in your apps.

I think the Temporal Web UI
* provides a central Web-based hub to inspect and manage Workflow Executions in Namespaces.

I think the Temporal CLI 
* provides a command-line based interface that mirrors many of the features available in the Temporal Web UI.

Reply:
Your thoughts aren't wrong, but for this page we want high-level simplicity. 
Our approachability research suggested that we need to avoid too many Temporal-centric phrases, terms, and explanations.

I agree with @jsundai 's small edits here - and I wouldn't expand beyond that atm.